### ISSUE TITLE
The python package should ship with model configs

### ISSUE THREAD

**Issue by jeanchristopheruel** [facebookresearch/segment-anything-2#51]
It would be interesting to simplify the package installation with  `python3 -m pip install git+https://github.com/facebookresearch/segment-anything-2.git`.

However, the user must currently provide the configuration file for the model given the search path for hydra, [which is set](https://github.com/facebookresearch/segment-anything-2/blob/86827e2fbae8a293f61d51caa70a4b0602c04454/sam2/__init__.py#L9C27-L9C39) at `pkg://sam2_configs`. Instead, I suggest including the model configs in the package installation.

**Comment by ronghanghu** [comment]
Hi @jeanchristopheruel, if you do
```
python3 -m pip install git+https://github.com/facebookresearch/segment-anything-2.git
```
it should install both `sam2` and `sam2_configs` packages into your environment, so that `sam2_configs` will also be in the search path for hydra

**Comment by ronghanghu** [comment]
This is resolved in #176. Now when running
```
python3 -m pip install git+https://github.com/facebookresearch/segment-anything-2.git
```
the SAM 2 config files would be copied over in the `sam2_configs` package added to user's site-packages
